core/base/slot_chain.go

```diff
@@ -125,15 +129,16 @@ func (sc *SlotChain) RangeStatPrepareSlot(f func(slot StatPrepareSlot)) {
 	}
 }
 
-func (sc *SlotChain) AddStatPrepareSlotFirst(s StatPrepareSlot) {
-	ns := make([]StatPrepareSlot, 0, len(sc.statPres)+1)
-	// add to first
-	ns = append(ns, s)
-	sc.statPres = append(ns, sc.statPres...)
-}
-
-func (sc *SlotChain) AddStatPrepareSlotLast(s StatPrepareSlot) {
+func (sc *SlotChain) InsertStatPrepareSlotByOrder(s StatPrepareSlot) {
 	sc.statPres = append(sc.statPres, s)
+	i := len(sc.statPres) - 1
```

Comment:
Can the standard pkg "sort" support sorting logic?
such as ‘sort.Slice()’？

Reply:
Refined.